### ISSUE TITLE
Release plans of pre-trained model without iterative transfer learning

### ISSUE THREAD

**Issue by setotaku0515** [DevashishPrasad/CascadeTabNet#151]
Hi,

Thank you for sharing your wonderful source code and pre-trained model.
Do you have any plans to release model weights which is trained by only one dataset, such as Tablebank and ICDAR2019?
Although, I know that iterative transfer learning using general dataset mentioned in the paper will improve accuracy, I would like to check that effect.

Best regards 👍
